### ISSUE TITLE
[Question] Load / Run bios (for cores that have BIOS with menu/options)

### ISSUE THREAD

**Issue by alaquepasa** [libretro/RetroArch#6952]
Some games of some actuals systems have system configuration dependent elements, like system time or language. This elements in retroarch are some times configurables in the core options menú, but in others there isn't.

For example, I in Reicast I can see "region" cnofiguration in core options, but not language, so I need to run the BIOS and go to system -> language. After this all games (that use this) are in my language (spanish).

To do this I need to use this bug: https://github.com/libretro/RetroArch/issues/6934 to load the BIOS (and not a game).

We can use this to handle memory cards too (like PSX if we use single memcard). For example, in Reicast I need to go to the BIOS and format the VM1 (and change the language)

So my question is: How can I load the BIOS without use this bug? And if this is not possible my request is have an option to load/run the core+bios.

**Comment by alaquepasa** [comment]
Yes, but you are ignoring the most important part: System configuration (+ systems that works with a integrated memcard, like Reicast, it doesn't split per game)

In reicast you can (and need) to select the languag in the BIOS, and I need to use a but to go to it.

Finally, if you have multiple VM you don't need this (at least I don't know why you need this) because you can copy/delete the files. So this will help for systems with config + people (and cores that only works) with single VM.

**Comment by Jj0YzL5nvJ** [comment]
I have not tried Reicast, but if it is about "the not intuitive" BIOS booting, I guess it will be similar to Mednafen PSX's cores.

Set 'Skip BIOS=Off' to can see the BIOS booting like in real hardware.

At boot time of any game:
F1 > Quick Menu > Disk Control > Disk Cycle Tray Status

The real question is, does Reicast have the CMOS emulation of the BIOS working properly? Dreamcast doesn't let you play games without setting the BIOS first...

**Comment by alaquepasa** [comment]
When you launch the first game it shows the date/time setting (only this) an then launch the game.

II diden't know that you can do this trick to launch BIOS, but Reicast doesn't have this "Disk Control" menu. But looking this I found in "Options" a "Launch to BIOS", if you set "ON" and then restart RetroArch it will launch the BIOS (wiithout "loading" the CD: You can't click "Game" in the menu, it say "Insert CD").

So now I can launch both BIOS, I think that they need a "common" options (for example, the Boot to BIOS from Reicast is fine, but I think that can be better with 3 values: OFF, ON and Siingle boot, the new is single boot that will go to OFF after next boot to BIOS)

I think that Mednafen (and others) need to have a common interface to enter the BIOS, for example that all cores with "useful" bios give the Reicast option "Boot to BIOS"

But I will close this because now I can load this 2 BIOS.

**Comment by fr500** [comment]
It's rather simple
Core have to set this env:
https://github.com/libretro/RetroArch/blob/master/libretro-common/include/libretro.h#L639

Then a case for `retro_load_game` with NULL content has to be implemented in the core.
That would result in what you want.

Actually you should re-open the issue, I think i's valid.

**Comment by orbea** [comment]
@fr500 This is not a core issue?

Take a look at 4DO where if you start it without content you get the bios, it might be better if reicast dropped the core option and did this too?
